# x,y,z and yaw angle for arm A with respect to origin
float64 arm_a_x_pos
float64 arm_a_y_pos
float64 arm_a_z_pos

# x,y,z and yaw angle for arm B with respect to origin
float64 arm_b_x_pos
float64 arm_b_y_pos
float64 arm_b_z_pos
---
